app/presenters/content_item_presenter.rb

```diff
@@ -159,11 +159,19 @@ def facets
         name: "Document type",
         preposition: "of type",
         type: "text",
-        display_as_result_metadata: true,
+        display_as_result_metadata: false,
         filterable: true,
         allowed_values: detailed_format_allowed_values,
       },
       {
+        key: "display_type",
+        name: "Document type",
+        preposition: "of type",
```

Comment:
`is_historic` is a bit of a weird case, but either way I think you're right about `type`. I don't think it's required.